forwards_paper.tex

```diff
@@ -1118,6 +1159,24 @@ \section{Simplification algorithm}
 
 \end{taocpalg}
 
+\newpage
+\clearpage
+\section*{Supplementary Information}
```

Comment:
since we're space unlimited (potentially) should we just put this as a second appendix rather than SI?